### PR TITLE
[charts/csi-unity] Centralize all images to one place

### PR DESCRIPTION
**Pull request by atye**
#### Is this a new chart?

No

#### What this PR does / why we need it:
Simplify images customization by centralizing them in one place

#### Which issue(s) is this PR associated with:

https://github.com/dell/csm/issues/1012

#### Special notes for your reviewer:
/!\ Only the syntax has been tested (passed with `helm install unity ./csi-unity -n unity --debug --dry-run
`) per agreement on Slack the Scrum team will validate the e2e installation.

#### Checklist:

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] Chart Version bumped
- [ ] Variables are documented in the chart README.md
- [x] Title of the PR starts with the chart name (e.g. `[charts_dir/mychartname]`) if applicable

### CODE REVIEW

**Review by cbartoszDell — DISMISSED** [review]

**Comment by atye** [comment]
Ensure that the images are up-to-date for 1.9.

**Comment by coulof** [comment]
> @coulof Ensure that the images are up-to-date for 1.9.

@atye, the sidecars are bumped to the latest version. I let the unity team decide on when to bump to v2.9.0

**Review by shanduur-dell — DISMISSED** [review]

**Review by gallacher — APPROVED** [review]

File: charts/csi-unity/values.yaml [hunk 1]
```diff
@@ -6,6 +6,20 @@
...
+  # CSI sidecars
+  attacher: registry.k8s.io/sig-storage/csi-attacher:v4.4.0
+  provisioner: registry.k8s.io/sig-storage/csi-provisioner:v3.6.0
+  snapshotter: registry.k8s.io/sig-storage/csi-snapshotter:v6.3.0
+  resizer: registry.k8s.io/sig-storage/csi-resizer:v1.9.0
+  registrar: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.9.0
+  healthmonitor: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.10.0
+
+  # CSM sidecars
+  podmon: dellemc/podmon:v1.7.0
```
Comment (shaynafinocchiaro) [inline charts/csi-unity/values.yaml:21]: Should podmon.image [here ](https://github.com/dell/helm-charts/blob/a35aebf2e32f5a83c0e560a45256dbad1fcf3d75/charts/csi-unity/values.yaml#L211) be removed in favor of this change?

Comment (coulof) [reply charts/csi-unity/values.yaml:21]: Good catch ! Yes, we need to remove that entry

**Review by coulof — CHANGES_REQUESTED** [review]

File: charts/csi-unity/values.yaml (hunk 1 above)
Comment (atye) [reply charts/csi-unity/values.yaml:21]: Fixed.

**Comment by atye** [comment]
@coulof Change is fixed for duplicate podmon image.

**Comment by shaynafinocchiaro** [comment]
There are duplicate entries for module images in isilon values too 
https://github.com/dell/helm-charts/blob/release-v1.9.0/charts/csi-isilon/values.yaml#L144
https://github.com/dell/helm-charts/blob/release-v1.9.0/charts/csi-isilon/values.yaml#L378

**Review by coulof — APPROVED** [review]
LGTM

**Comment by atye** [comment]
> There are duplicate entries for module images in isilon values too https://github.com/dell/helm-charts/blob/release-v1.9.0/charts/csi-isilon/values.yaml#L144 https://github.com/dell/helm-charts/blob/release-v1.9.0/charts/csi-isilon/values.yaml#L378

Addressed in https://github.com/dell/helm-charts/pull/320.

**Review by shaynafinocchiaro — APPROVED** [review]

**Pull request merged**